src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.module.css

```diff
@@ -0,0 +1,63 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 30px 1fr 30px;
+    grid-template-rows: auto;
+  }
+}
+
+.carouselWrapper {
+  grid-column: 2;
+}
+
+@media (--tablet-and-down) {
+  .carouselWrapper {
+    column-gap: 10px;
+    grid-column: 1/ span 13;
+    grid-column: 1 / span 13;
+    grid-template-columns: repeat(13, minmax(0, 1fr));
+    padding: 0;
+  }
+}
+
+.slideQuote {
+  background: var(--color-gray-300);
```

Comment:
👍 Whoa! What is this magic? Thanks for introducing box-sizing property.

Reply:
Yeah, it's a bit magical. It's a property that changes the definition of "width" and "height". Some people turn it on for all elements, since they find it easier to style.